<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>굴림체</family>
         </font>
        </property>
        <property name="text">
         <string>ASIS논리/물리명</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineASISTableEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>굴림체</family>
         </font>
        </property>
        <property name="text">
         <string>ASIS속성/컬럼명</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineASISColumnEdit"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QPushButton" name="pushSearchButton">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>굴림체</family>
       </font>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>굴림체</family>
         </font>
        </property>
        <property name="text">
         <string>TOBE논리/물리명</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineTOBETableEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>굴림체</family>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>TOBE속성/컬럼명</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineTOBEColumnEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <family>굴림체</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>굴림체</family>
       </font>
      </property>
      <property name="text">
       <string>컬럼 매핑정보 반영</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushInputButton">
        <property name="font">
         <font>
          <family>굴림체</family>
         </font>
        </property>
        <property name="text">
         <string>매핑정보 엑셀 파일 입력/수정</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushSampleButton">
        <property name="font">
         <font>
          <family>굴림체</family>
         </font>
        </property>
        <property name="text">
         <string>샘플서식 다운로드</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
